libvast/test/pipeline.cpp

```diff
@@ -179,6 +179,28 @@ TEST(tail 5) {
   }
 }
 
+TEST(unique) {
+  auto ops = unbox(pipeline::parse("select id.orig_h | unique")).unwrap();
+  ops.insert(ops.begin(), std::make_unique<source>(std::vector<table_slice>{
+                            head(zeek_conn_log.at(0), 1), // = 1
+                            head(zeek_conn_log.at(0), 1), // + 0
+                            head(zeek_conn_log.at(0), 5), // + 4
+                            head(zeek_conn_log.at(0), 0), // + 0
+                            head(zeek_conn_log.at(0), 5), // + 4
+                            head(zeek_conn_log.at(0), 7), // + 5
+                            head(zeek_conn_log.at(0), 7), // + 6
```

Comment:
Please also implement this test within one table slice rather than just at the boundaries of slices, e.g., by calling `concatenate(...)` on the same data set.

Reply:
The last slice has a duplicate within itself, but I added another slice to cover the case where the duplicate is positioned differently.